### ISSUE TITLE
[Governance] Move plugin to nvim-lua?

### ISSUE THREAD

**Issue by VVKot** [oberblastmeister/neuron.nvim#52]
Given the overall high quality of the plugin, its stability, and low activity in recent months, I wonder if it makes sense to move it to https://github.com/nvim-lua org? We'd benefit from the larger Lua expertise, more devs to review/merge PRs, and additional promotion for the plugin.

**Comment by VVKot** [comment]
@oberblastmeister care to weight in here?

**Comment by oberblastmeister** [comment]
I don't think it makes sense to do that. There are many good plugins but I don't think that they should be moved to nvim-lua just because. I think it should be more for 'core' plugins like plenary that are meant to be used in other plugins. Even telescope is not in nvim-lua
